<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-31T12:45:36 IST" name="Suite" finished-at="2024-12-31T12:45:43 IST" duration-ms="6818">
    <groups>
      <group name="ErorrHandling">
        <method signature="ErrorValidations.sumbitReuest()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" name="sumbitReuest" class="Sel.TestCases.ErrorValidations"/>
      </group> <!-- ErorrHandling -->
    </groups>
    <test started-at="2024-12-31T12:45:36 IST" name="Submit Order" finished-at="2024-12-31T12:45:36 IST" duration-ms="5">
    </test> <!-- Submit Order -->
    <test started-at="2024-12-31T12:45:37 IST" name="Eror Validation" finished-at="2024-12-31T12:45:43 IST" duration-ms="6638">
      <class name="Sel.TestCases.ErrorValidations">
        <test-method is-config="true" signature="lauchBrowser()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:37 IST" name="lauchBrowser" finished-at="2024-12-31T12:45:39 IST" duration-ms="2259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchBrowser -->
        <test-method retried="true" signature="sumbitReuest()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:39 IST" name="sumbitReuest" finished-at="2024-12-31T12:45:39 IST" duration-ms="229" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Incorrect email or password.> but was:<Incorrect email  password.>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Incorrect email or password.> but was:<Incorrect email  password.>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at Sel.TestCases.ErrorValidations.sumbitReuest(ErrorValidations.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumbitReuest -->
        <test-method is-config="true" signature="closingBrowser()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:39 IST" name="closingBrowser" finished-at="2024-12-31T12:45:39 IST" duration-ms="112" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingBrowser -->
        <test-method is-config="true" signature="lauchBrowser()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:39 IST" name="lauchBrowser" finished-at="2024-12-31T12:45:42 IST" duration-ms="2500" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- lauchBrowser -->
        <test-method signature="sumbitReuest()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:42 IST" name="sumbitReuest" finished-at="2024-12-31T12:45:43 IST" duration-ms="1188" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<Incorrect email or password.> but was:<Incorrect email  password.>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<Incorrect email or password.> but was:<Incorrect email  password.>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at Sel.TestCases.ErrorValidations.sumbitReuest(ErrorValidations.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumbitReuest -->
        <test-method is-config="true" signature="closingBrowser()[pri:0, instance:Sel.TestCases.ErrorValidations@350aac89]" started-at="2024-12-31T12:45:43 IST" name="closingBrowser" finished-at="2024-12-31T12:45:43 IST" duration-ms="108" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingBrowser -->
      </class> <!-- Sel.TestCases.ErrorValidations -->
    </test> <!-- Eror Validation -->
  </suite> <!-- Suite -->
</testng-results>
